### PR TITLE
Update registry landing page.md

### PR DESCRIPTION
**Pull request by noaleetz**
still need to fix code snippet

## Description

What does the pull request do? If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/registry/intro.md [hunk 1]
```diff
@@ -9,20 +9,61 @@ displayed_sidebar: default
...
+# [Log](https://docs.wandb.ai/ref/python/run#log_artifact) and [link](https://docs.wandb.ai/ref/python/run#link_artifact) the model to the Model Registry
+run.log_artifact(path="./my_model.h5", name="gemma-finetuned-3twsov9e")
+run.link_artifact(artifact="gemma-finetuned-3twsov9e", target_path="<YOUR_ORG_NAME>/wandb-registry-model/registry-quickstart-collection",
+
+run.finish()
+```
+
+1. When working with models, connect transitions to CI/CD workflows: transition candidate models through workflow stages and [automate downstream actions](https://docs.wandb.ai/guides/model_registry/model-registry-automations) with webhooks or jobs.
+
+## How to get started
```
Comment (ngrayluna) [inline docs/guides/registry/intro.md:54]: I think linking to resources that show the old Model Registry will confuse users. Let's link to resources that show the new UI, emphasize the new user story, etc.

File: docs/guides/registry/intro.md [hunk 2]
```diff
@@ -9,20 +9,61 @@ displayed_sidebar: default
...
+- [Automate](https://docs.wandb.ai/guides/model_registry/model-registry-automations) downstream processes and model CI/CD.
+- Track an [artifact’s lineage](https://docs.wandb.ai/guides/model_registry/model-lineage) and audit the history of changes to production artifacts.
+- Configure viewer, member, or admin access to a registry for all org users
 
 ## How it works
-W&B Registry is composed of three main components: registries, collections, and [artifact versions](../artifacts/create-a-new-artifact-version.md).
 
-A *registry* is a repository or catalog for ML assets of the same kind. You can think of a registry as the top most level of a directory. Each registry consists of one or more sub directories called collections. A *collection* is a folder or a set of linked [artifact versions](../artifacts/create-a-new-artifact-version.md) inside a registry. An [*artifact version*](../artifacts/create-a-new-artifact-version.md) is a single, immutable snapshot of an artifact at a particular stage of its development.  A registry belongs to an organization, not a specific team.
+Track and manage your staged artifacts with a few simple steps.
```
Comment (ngrayluna) [inline docs/guides/registry/intro.md]: Can we remove the word "staged"? Perhaps something like?: 

"Track and manage artifacts in a registry with a few simple steps"

File: docs/guides/registry/intro.md (hunk 1 above)
Comment (noaleetz) [reply docs/guides/registry/intro.md:54]: Until we have the equivalent docs for registry (those are the docs that I'm suggesting we include in the post-MVP epic) my thought was that we should still link to some resource in the meantime even though it is specific to model registry. We can remove but if so would like to align on potential timeline for when we can add them back so users know about these capabilities. Let's discuss more in [this](https://weightsandbiases.slack.com/archives/C05UUJ350CA/p1719525895678949?thread_ts=1719514599.925499&cid=C05UUJ350CA) thread

Comment (noaleetz) [reply docs/guides/registry/intro.md:54]: for example we need to be able to link to automations for a registry

File: docs/guides/registry/intro.md (hunk 2 above)
Comment (noaleetz) [reply docs/guides/registry/intro.md]: are you worried staged will intro a new potential definition? Can we do "Manage your artifacts ready for staging in a few steps"

**Comment by ngrayluna** [comment]
Missed this during the first review, let's change the wording on this:

"Look out for incoming information on a migration we will be making available to migrate contents from the current model registry into the new model registry inside W&B Registry. You can reach out to support@wandb.com with any questions or to speak to our product team about any concerns with the migration."

Things to change:
1. Don't use language that dates the docs "Look out for...."
2. Refer to W&B not"our", "we", etc. 
3. This "..or to speak to our product team about any concerns with the migration." won't make sense for non beta users. Can we remove it? Or provide an email? (Or other point of contact)
4. We'll want to be careful on product capitalization. So model registry should be capitalized in "...from the current model registry into the new model registry inside W&B Registry."

**Review by ngrayluna — CHANGES_REQUESTED** [review]
Image HTML tag is breaking the docs: https://weightsandbiases.slack.com/archives/C01DX8LRZEJ/p1719863302819619?thread_ts=1719861040.785199&cid=C01DX8LRZEJ

**Comment by ngrayluna** [comment]
Fixed images and links. Condensed intro paragraph and tried to make it less market-y and more geared to devs. Current preview here:
[Registry _ Weights & Biases Documentation.pdf](https://github.com/user-attachments/files/16074208/Registry._.Weights.Biases.Documentation.pdf)

**Comment by ngrayluna** [comment]
@noaleetz Lemme know if this looks good. If yay, I'll merge :)

**Review by ngrayluna — APPROVED** [review]

**Pull request merged**